src/Symfony/Component/Messenger/Envelope.php

```diff
@@ -48,22 +48,26 @@ public function with(StampInterface ...$stamps): self
         $cloned = clone $this;
 
         foreach ($stamps as $stamp) {
```

Comment:
while at it.. can we update with() to take at least one stamp?

Reply:
It works with no stamp so maybe not. Eg array_merge() is annoying to use with the coalesce operator for this reason.

Reply:
hm i see, we're just make the behavior more lax with e.g. `with($somethingComputed)`